activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -199,10 +187,15 @@ def deserialize_entry(raw_value)
           if raw_value
             entry = Marshal.load(raw_value) rescue raw_value
             entry.is_a?(Entry) ? entry : Entry.new(entry)
-          else
-            nil
           end
         end
+
+        def rescue_error(fallback)
```

Comment:
I renamed it to `rescue_errors_with nil` to not use an extra keyword arg / allocate a hash / be simple ... looks good ?